webpack.config.js

```diff
@@ -0,0 +1,152 @@
+const path = require('path');
+
+const assetDir  = './core/';
+const outputDir = 'core/dist';
```

Comment:
How about we put this at the root of the directory so that this isn't nested?

Reply:
eg: `const outputDir = 'dist';`

Reply:
My thought was to have everything someone building on top of decanter might need contained in one directory. We had settled on `core` a while ago for consistency with other npm packages, specifically bourbon. If people want to cherry pick components, they can find source in `core/scss` and `core/js`; if they just want to use the prebuilt assets, they can find them in `core/dist`.

I suppose this breaks down a bit because people building on decanter might want to use the twig templates, which are not in `core/`. But my goal was for folks to be able to everything they need to build with in `core/`.

Reply:
I think I am leaning more towards the `src/dist` model these days. I've seen that pattern in a number of packages as well and seems to be the module most webpack projects use. 

https://stackoverflow.com/questions/23730882/what-is-the-role-of-src-and-dist-folders/23731040#23731040

https://guide.freecodecamp.org/miscellaneous/react-with-webpack-file-structure/

We can add a post-build step to move the twig files into place if we feel that is neccessary but those files, along with the sass, feel like source files to me. I think at some point we will get into a javascript framework where the templates would be outputted to this dist directory.

@yvonnetangsu @josephgknox thoughts?

Reply:
Hmm.... I do like the idea of having separate src/dist directories because there's no ambiguity and that the asset folders (js, css/scss) have the same depths in the nested structure. In this model, where would the img folder go? Twig files seems like they belong to both. I also like the idea of someone developing on top of decanter can find everything they need in one directory. 

I see that @sherakama just pushed up an alt branch - I'll check it out.